techs/enb-postcss.js

```diff
@@ -3,7 +3,7 @@ var vow     = require('vow'),
     postcss = require('postcss'),
     pimport = require('postcss-import');
 
-module.exports = require('enb/techs/css').buildFlow()
+module.exports = require('enb/lib/build-flow').create()
```

Comment:
If you need support `enb@1.x` and `enb@0.x`, then you can use `enb.buildFlow || require('enb/lib/build-flow')`.